srtcore/crypto.cpp

```diff
@@ -449,7 +449,7 @@ void CCryptoControl::sendKeysToPeer(Whether2RegenKm regen SRT_ATR_UNUSED)
                 HLOGC(cnlog.Debug, log << "sendKeysToPeer: SENDING ki=" << ki << " len=" << m_SndKmMsg[ki].MsgLen
                         << " retry(updated)=" << m_SndKmMsg[ki].iPeerRetry);
                 m_SndKmLastTime = now;
-                m_parent->sendSrtMsg(SRT_CMD_KMREQ, (uint32_t *)m_SndKmMsg[ki].Msg, m_SndKmMsg[ki].MsgLen/sizeof(uint32_t));
+                m_parent->sendSrtMsg(SRT_CMD_KMREQ, (uint32_t *)m_SndKmMsg[ki].Msg, m_SndKmMsg[ki].MsgLen / sizeof(uint32_t));
```

Comment:
Would be clearer, if you split the line at the argument separator.

Reply:
Not sure I get the suggestion. Like this?
```c++
m_parent->sendSrtMsg(
    SRT_CMD_KMREQ, (uint32_t *)m_SndKmMsg[ki].Msg, m_SndKmMsg[ki].MsgLen / sizeof(uint32_t));
```

Anyway, I believe this change is not that critical. There is a pending global clang-formatting of the code.

Reply:
No, I meant like this:
```c++
m_parent->sendSrtMsg(SRT_CMD_KMREQ, (uint32_t *)m_SndKmMsg[ki].Msg,
                                       m_SndKmMsg[ki].MsgLen / sizeof(uint32_t));
```

Not critical, of course, although I'm worried a bit about this clang-formatting. :)

Reply:
> although I'm worried a bit about this clang-formatting. :)

Yeap, it might be not that good, as manual formatting. But will see. Maybe at least some partial formatting.